command/sub8_launch/launch/subsystems/online_bagger.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node pkg="mil_tools" type="online_bagger.py" name="online_bagger" output="screen"/>
+    <rosparam file="$(find sub8_launch)/config/online_bagger.yaml" command="load" ns="online_bagger"/>
+</launch>
```

Comment:
All of @ironmig's shiny new ROS parameters should be added to this launch file. Make sure to leave the bag storage directory blank so it defaults to using the `BAG_DIR` environment variable.